src/ansys/fluent/core/services/settings.py

```diff
@@ -249,30 +229,37 @@ def _extract_static_info(self, info):
             ret["object-type"] = self._extract_static_info(info.object_type)
         if info.help:
             ret["help"] = info.help
-        return ret
 
-    @_trace
-    def get_obj_static_info(self):
-        request = SettingsModule.GetObjectStaticInfoRequest()
-        request.root = "fluent"
-        response = self.__service_impl.get_obj_static_info(request)
+        try:
+            if info.include_child_named_objects:
+                ret["include_child_named_objects"] = info.include_child_named_objects
+        except AttributeError:
+            pass
 
-        return self._extract_info(response.info)
+        try:
+            if info.list_size:
+                ret["list_size"] = info.list_size
```

Comment:
@V-Dandekar Does ret["list_size"] get set to 0 here for settings which are not fixed size?

Reply:
no.